platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -62,59 +65,64 @@ public static PlatformSchedulers create(@NonNull final PlatformContext context,
                         .withFlushingEnabled(true)
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
                         .cast(),
                 model.schedulerBuilder("orphanBuffer")
-                        .withType(config.getOrphanBufferSchedulerType())
+                        .withType(config.orphanBufferSchedulerType())
                         .withUnhandledTaskCapacity(config.orphanBufferUnhandledCapacity())
                         .withFlushingEnabled(true)
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
                         .cast(),
                 model.schedulerBuilder("inOrderLinker")
-                        .withType(config.getInOrderLinkerSchedulerType())
+                        .withType(config.inOrderLinkerSchedulerType())
                         .withUnhandledTaskCapacity(config.inOrderLinkerUnhandledCapacity())
                         .withFlushingEnabled(true)
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
                         .cast(),
                 model.schedulerBuilder("linkedEventIntake")
-                        .withType(config.getLinkedEventIntakeSchedulerType())
+                        .withType(config.linkedEventIntakeSchedulerType())
                         .withUnhandledTaskCapacity(config.linkedEventIntakeUnhandledCapacity())
                         .withFlushingEnabled(true)
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
                         .cast(),
                 model.schedulerBuilder("eventCreationManager")
-                        .withType(config.getEventCreationManagerSchedulerType())
+                        .withType(config.eventCreationManagerSchedulerType())
                         .withUnhandledTaskCapacity(config.eventCreationManagerUnhandledCapacity())
                         .withFlushingEnabled(true)
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
                         .cast(),
                 model.schedulerBuilder("signedStateFileManager")
-                        .withType(config.getSignedStateFileManagerSchedulerType())
+                        .withType(config.signedStateFileManagerSchedulerType())
                         .withUnhandledTaskCapacity(config.signedStateFileManagerUnhandledCapacity())
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
+                        .cast(),
+                model.schedulerBuilder("stateSigner")
```

Comment:
Did you intentionally not enable metrics for this scheduler?

Reply:
its a direct call, so there is no unhandled capacity